scripts/auto_restart_devnet.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -e
+
+pushd $(dirname "$0")/..
+
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "** Interrupted by CTRL-C"
+    DEVNET_PID=$(pgrep devnet)
+    echo "Devnet PID is $DEVNET_PID. Killing it"
+    kill -9 $DEVNET_PID
+}
+
+while :
+do
+    echo "Starting the devnet"
+    cargo run --release --package=devnet -- --log-level Debug --test-block-period=500 >> devnet.log 2>&1 &
+    sleep 1
+    until scripts/rpc.py view_account -a alice.near
+    do
+        echo "The devnet is not stated yet"
+        sleep 1
+    done
+    ACCOUNT=rebooter_$(openssl rand -hex 10)
+    echo "Creating account $ACCOUNT"
+    scripts/rpc.py create_account $ACCOUNT 0
+    sleep 3
+    CONTRACT=rebooter_$(openssl rand -hex 10)
+    echo "Creating contract $CONTRACT"
+    scripts/rpc.py create_account -s $ACCOUNT $CONTRACT 0
+    sleep 3
+    echo "Deploying code at tests/hello.wasm"
+    scripts/rpc.py deploy -s $ACCOUNT $CONTRACT tests/hello.wasm
+    sleep 3
+    while :
+    do
+        VALUE=$(openssl rand -hex 10)
+        echo "Setting value $VALUE"
+        scripts/rpc.py schedule_function_call -s $ACCOUNT $CONTRACT setValue --args=\{\"value\":\"$VALUE\"\}
+        sleep 5
```

Comment:
I would be more conservative with delays and put them into overridable env vars

Reply:
I think that delay of 5 sec is cool. Maybe we should increase some setup delays